webservices/tasks/__init__.py

```diff
@@ -29,8 +29,12 @@ def redis_url():
     CELERYBEAT_SCHEDULE={
         'refresh': {
             'task': 'webservices.tasks.refresh.refresh',
-            'schedule': crontab(minute=0, hour=9),
+            'schedule': crontab(minute=0, hour=9, day_of_week='sun-fri'),
         },
+        'refresh_all': {
+            'task': 'webservices.tasks.refresh.refresh_all',
+            'schedule': crontab(minute=0, hour=9, day_of_week='sat'),
```

Comment:
Just to make sure I have my times right, this will be starting at 9 AM UTC (http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#time-zones), correct?